brainstormforce/ultimate-addons-for-gutenberg
# Fix: Container - Resolved warning for backgroundVideoOpacity.

### Description

- Warning - https://share.getcloudapp.com/QwuLzZ7A

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Use this code - https://gist.github.com/Yodapp/8482893f4e9cc97de58ca5642dddb882
- Publish the page and check issue is resolved or not also check to debug log.

### Checklist:
